src/modules/wrap-with-usecallback.ts

```diff
@@ -0,0 +1,61 @@
+import { selectedText, importMissingDependencies, activeFileName, allText, selectedTextStart, selectedTextEnd } from "../editor";
+import { replaceSelectionWith } from "../code-actions";
+import { buildEffectHook, buildUseCallbackHook } from "../snippet-builder";
+import { codeToAst, astToCode } from "../parsing";
+import { persistFileSystemChanges } from "../file-system";
+import * as t from '@babel/types';
+import { pathContains } from "../ast-helpers";
+import traverse from "@babel/traverse";
+
+
+function isFunctionExpression(path) {
+    return t.isVariableDeclaration(path) || t.isFunctionExpression(path.node.init)
+}
+
+function isFunction(path) {
+    return (t.isFunctionDeclaration(path) || isFunctionExpression(path))
+}
+
+
+export function findSelectedFunctionInRange(ast, start, end) {
+    let foundPath = null;
+    const visitor = {
+        'VariableDeclaration|FunctionDeclaration'(path) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'path' to have a typedef](https://app.codacy.com/gh/wix/vscode-glean/pullRequest?prid=5535229)